index.html

```diff
@@ -189,30 +207,28 @@ <h2 class="presentation2">About <br> myself <br></h2>
 </section>
 <!-- about myself finish-->
   <!-- contact form start -->
-<section class="sec3">
+<section class="sec3" id="contact">
       <div class="contact">
           <div class="contact_text">
           <h2 class="contactme">Contact me</h2>
           <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
           </div>
-          <form action="https://formspree.io/f/xnqwodoq" method="POST">
+          <div class="form">
+           <form action="https://formspree.io/f/xnqwodoq" method="POST">
               <div class="input">
                 <input type="text" name="name" maxlength="30" placeholder="Yeremias" required>
               </div>
               <div class="input">
                 <input type="email" name="email" placeholder="natajayanji14@gmail.com" required>
-              </div>
-              
-              
+              </div>              
               <div class="input">
-                <textarea name="message" cols="30" rows="10" maxlength="500" placeholder="Write in your message here" required></textarea>
-              </div>
-              
-             
-              
+                <textarea name="message" cols="30" rows="10" maxlength="500" placeholder="Write your message here" required></textarea>
+              </div>           
               <button class="button_touch" type="submit">Get In Touch</button>
           </form>
+        </div>
       </div>
   </section>
```

Comment:
- [ ] Hey, Please also kindly make sure your contact section dominates your body background here also. check the images to compare so you can fix the issue with your design compared to that of the Figma design.

:point_down: **YOUR DESIGN**

![contact](https://user-images.githubusercontent.com/59397159/165690782-5100cf8b-4810-4633-aab5-9d7a85fc61ac.png)



:point_down: **FIGMA DESIGN**

![figCont](https://user-images.githubusercontent.com/59397159/165690875-dee3d53c-0422-4804-88f5-ede01e3b2ca0.png)